### PR TITLE
Add ros server

### PR DESCRIPTION
**Pull request by enriquea52**
Added cmake instruction
Added overlead of insertPointCloud() to directly accept PCL pointclouds
Corrected bonxai_ros/README.md file

### CODE REVIEW

**Comment by enriquea52** [comment]
This changes are enough for compilation and execution

File: bonxai_map/include/bonxai_map/probabilistic_map.hpp [hunk 1]
```diff
@@ -162,6 +167,23 @@ inline void ProbabilisticMap::insertPointCloud(const std::vector<PointT>& points
...
+  const double max_range_sqr = max_range*max_range;
+  for(const auto& point: points)
+  {
+    Eigen::Vector3f tof = point.getVector3fMap();
+    const auto to = tof.cast<double>();
+    addPoint(from, to, max_range, max_range_sqr);
+  }
+  updateFreeCells(from);
+}
+
```
Comment (facontidavide) [inline bonxai_map/include/bonxai_map/probabilistic_map.hpp:186]: No need.
change the above version to:

```c++
template<typename PointT, typename Alloc> inline
void ProbabilisticMap::insertPointCloud(const std::vector<PointT,  Alloc>  &points,
                                        const PointT &origin,
                                        double max_range)
```

File: bonxai_ros/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ cd ~/ws_bonxai/src
 ```
 2. Git clone the Bonxai package
 ```bash
-git clone https://github.com/facontidavide/Bonxai.git -b ros2
+git clone https://github.com/facontidavide/Bonxai.git -b add_ros_server
```
Comment (facontidavide) [inline bonxai_ros/README.md:11]:
```suggestion
git clone https://github.com/facontidavide/Bonxai.git
```

**Review by facontidavide — CHANGES_REQUESTED** [review]

**Comment by facontidavide** [comment]
I decided to cherry pick the changes. thanks

**Pull request closed without merging**